src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -80,21 +84,25 @@ private Rect GetBoundingClientRect()
 			return new Rect(double.Parse(sizeParts[0]), double.Parse(sizeParts[1]), double.Parse(sizeParts[2]), double.Parse(sizeParts[3]));
 		}
 
-		public UIElement(string htmlTag = "div", bool isSvg = false)
+		public UIElement() : this(null, false) { }
+
+		public UIElement(string htmlTag = DefaultHtmlTag) : this(htmlTag, false) { }
```

Comment:
Codacy found an issue: [This method signature overlaps the one defined on line 87, the default parameter value can't be used.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)